### ISSUE TITLE
Consider adding `wasm32-unknown-unknown` to standalone install.

### ISSUE THREAD

**Issue by follower** [rust-lang/rust#77503]
### Can't install targets with standalone installer

With a [standalone install](https://forge.rust-lang.org/infra/other-installation-methods.html#standalone-installers) of Rust (e.g. via [`rust-1.46.0-x86_64-apple-darwin.pkg`](https://static.rust-lang.org/dist/rust-1.46.0-x86_64-apple-darwin.pkg)) it is not possible to "add" additional targets (specifically, the required precompiled `std` etc).

A reason for this is that `rustup` does not support being run when a standalone version is installed (see: https://github.com/rust-lang/rust/issues/64550).

(Maybe this means a feature request for `rustup` might be better to support target installation when run with a standalone install.)

### Is WASM a special case?

While it seems reasonable to not include *all* targets in the standalone install, it seems like there might be some argument for WASM to be treated differently due to potentially (I have no sample data aside from me :) ) being the most likely cross platform target a person using a standalone install might want.

### Or just document the manual approach?

An alternative approach would be to officially document how to locate & install other targets (and/or `wasm32-unknown-unknown` specifically) somewhere.

I discovered it is possible to download and install the target packages manually and am partway through documenting the process here:

 * <http://www.labradoc.com/i/follower/p/notes-rust-language#20201001>

**TL;DR:**

 1) Download `https://static.rust-lang.org/dist/rust-std-1.42.0-wasm32-unknown-unknown.tar.gz` (with version modified as appropriate). (Discover other versions via e.g. `http://static.rust-lang.org/dist/channel-rust-stable.toml` as documented at https://forge.rust-lang.org/infra/channel-layout.html.)

 2) Run the installer script.

**Comment by jonas-schievink** [comment]
This already seems documented in the [standalone installers](https://forge.rust-lang.org/infra/other-installation-methods.html#standalone-installers) page:

> These installers come with rustc, cargo, rustdoc, the standard library, and the standard documentation, but do not provide access to additional cross-targets like rustup does.

I don't think we should promote wasm like that, users can just use rustup if they want to target it.

**Comment by Mark-Simulacrum** [comment]
I think adding a note to that page about how to add targets manually afterwards shouldn't be too hard, and would be happy to review a PR to that effect, but I agree with @jonas-schievink that we do not want to special case wasm (or more broadly add any additional targets to the standalone installers).
